apps/nextjs-website/src/_contents/translations.ts

```diff
@@ -333,11 +333,14 @@ export const translations = {
       noAccount: 'Non hai un account?',
     },
     confirmLogin: {
-      title: 'Verifica di accesso MFA',
-      body: 'Inserisci il codice di verifica inviato nella tua e-mail',
-      code: '012345',
-      wrongAccount: 'Hai sbagliato account? ',
-      send: 'Invia',
+      title: 'Verifica la tua identità',
+      body: (email: string) =>
+        `Abbiamo inviato un codice di verifica a <b>${email}</b><br>Il codice scade tra 30 minuti.`,
```

Comment:
```suggestion
        `Abbiamo inviato un codice di verifica a <b>${email}</b><br>Il codice scade tra 3 minuti.`,
```
The minutes value should be an env variable i had this issue to solve the problem https://pagopa.atlassian.net/browse/DEV-1182